systest/backup/nfs-backup/docker-compose.yml

```diff
@@ -0,0 +1,377 @@
+
+version: "3.5"
+services:
+#HA backup cluster
+  alpha1:
+    image: dgraph-nfs-client
+    working_dir: /data/alpha1
+    links:
+      - "nfs:nfs"
+    privileged: true
+    labels:
+      cluster: test
+    ports:
+    - "8080"
+    - "9080"
+    volumes:
+    - type: bind
+      source: $GOPATH/bin
+      target: /gobin
+      read_only: true
+
+    - type: bind
+      source: ../../../tlstest/mtls_internal/tls/alpha1
+      target: /dgraph-tls
+      read_only: true
+    command: /bin/bash  /usr/local/bin/rundgraph.sh  alpha1 zero1:5080,zero2:5080,zero3:5080
+  alpha2:
```

Comment:
we should improve upon the naming convention so that it is clear what alpha belongs to what cluster.

Reply:
done @aman-bansal